drivers/gpio/Kconfig.xlnx

```diff
@@ -7,5 +7,6 @@ config GPIO_XLNX_AXI
 	bool "Xilinx AXI GPIO driver"
 	default y
 	depends on DT_HAS_XLNX_XPS_GPIO_1_00_A_ENABLED
+	select PINCTRL
```

Comment:
This driver does not use pinctrl (pins are routed in the FPGA design, there is no pin controller associated).

Reply:
Removed